papis/commands/addto.py

```diff
@@ -78,6 +78,8 @@ def run(document, filepaths):
         )
         shutil.copy(in_file_path, endDocumentPath)
 
+    if not "files" in document.keys():
+        document["files"] = []
     document['files'] += new_file_list
```

Comment:
I would rather see `document["files"] = document.getdefault("files", []) + new_file_list`

Reply:
I'm not quite sure what you mean. I see that the `Document` object has a method called `get_files` (but I'm not seeing anything like `getdefault`). The issue with `get_files` is that it returns the `files` list with full file paths. I'm using a papis library across macOS and linux and thus only want to store relative file paths in `files`.

After re-looking at `document.py` I'm questioning why this is even necessary.  The `__setitem__` method adds the key if its not present. Will try to reproduce and re-debug.